src/components/Group/Group.js

```diff
@@ -5,85 +5,96 @@ import Task from '../Task/Task';
 import MyButton from '../ui/button/MyButton';
 import styles from './Group.module.css';
 import MyModal from '../ui/modal/MyModal';
+import { useDispatch } from 'react-redux';
+import { fetchDeleteTask, fetchUpdateGroup, fetchUpdateTask } from '../../redux/slices/groups';
 
 const ListGroup = ({
-  group, 
-  groups, 
-  setGroups, 
+  group,
+  setStatistics,
   removeGroup, 
 }) => {
+    const groupId = group._id;
+    const dispatch = useDispatch();
     const [edit, setEdit] = useState(null);
     const [value, setValue] = useState('');
     const [modalTaskVisible, setModalTaskVisible] = useState(false);
-    const checkCompleted = () => {
+    const checkCompleted = (group) => {
         let completed = 0;
         let notCompleted = 0;
         group.tasks.map((i) => i.status ? ++notCompleted : ++completed);
```

Comment:
After destructuring object:
```suggestion
        tasks.map((i) => i.status ? ++notCompleted : ++completed);
```